ballista/rust/core/proto/ballista.proto

```diff
@@ -408,6 +421,119 @@ message PhysicalPlanNode {
   }
 }
 
+// physical expressions
```

Comment:
Physical expressions are now serialized and deserialized without involving logical expressions. It is needed because logical columns and physical columns are represented differently, one with relation qualifier and the other one with column index. This should also speed up the deserialization for physical plane because we won't need to run physical expressions through physical planner anymore.

Reply:
Yeah, i have the same feeling when it comes to debugging index offset issues :(